JuliaGPU/session2/Feyaz/qn2.jl

```diff
@@ -0,0 +1,58 @@
+using Test
+
+#rootinator func takes the roots.
+
+function rootinator(num)
+	start=0
+	fin=num
+	ans=0
+	#step=0
+	while start<=fin
+		mid= trunc(Int, (start+fin)/2)
+		if(mid*mid==num)
+			ans=mid
+			break
+		end
+		if (mid*mid<num)
+			start=mid+1
+			ans=mid
+		else
+			fin=mid-1
+		end
+		#println(step)
+		#step=step+1
+	end
+	inc=0.1
+	prec=15#for Int64, ig
```

Comment:
```suggestion
	prec=7#for Int64, ig
```
precision = 15 takes too long for large numbers.

Reply:
Done